extensionblocker.js

```diff
@@ -0,0 +1,38 @@
+// Note: jQuery needed
+var result;
+var allowedExtensions = [];
+function checkExt(name,codesite,downloadsite) {
+var datavar;
+$.get(codesite,function(data,status){datavar=data;
+// a code idea for code-based filters
+if (datavar.search("post")===-1) {
+// other filters like this would go inside here and the following code would go inside all of them
+var website;
+$.get("view-source:"+downloadsite,function(data,status){website=data;
```

Comment:
Indent by four spaces, change `"view-source:"` to `'view-source:'`, add space after "function", remove or rename "status" to `_`